docs/usage.md

```diff
@@ -81,15 +81,25 @@ cd <repo root dir>/docs/examples
 make clean
 ```
 
-### Log your build
+### Generate a compilation database
+
+There are different ways to generate a 
+[compilation database](https://clang.llvm.org/docs/JSONCompilationDatabase.html).
+This compilation database is an input of the next analysis step.
+
+You can check [LLVM documentation](https://clang.llvm.org/docs/JSONCompilationDatabase.html)
+for the different options on how to generate this file.
+
+Alternatively, you can run `CodeChecker log` command to generate a compilation database.
+
 Logging means that during the whole compilation process CodeChecker catches
-compiler calls and saves commands in a
-[compilation database](https://clang.llvm.org/docs/JSONCompilationDatabase.html)
-file. This compilation database is an input of the next analysis step.
+compiler calls and saves commands in a compilation database file. 
 
 note: This command is currently not working on macOS.
 The System Integrity Protection (SIP) in macOS blocks the usage of `LD_LIBRARY_PATH`.
-Please try to obtain the `compile_commands.json` in an other way, with the help of CMake for example.
+Please try to obtain the `compile_commands.json` in an other way, with the help of
+[CMake](https://cmake.org/cmake/help/latest/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html)
+for example.
```

Comment:
So maybe I should close this PR and open a new one only keeping this last change? (link to CMake documentation), what do you think?